src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpExporterOptionsExtensions.cs

```diff
@@ -0,0 +1,182 @@
+// <copyright file="OtlpExporterOptionsExtensions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using Grpc.Core;
+using OpenTelemetry.Exporter.OpenTelemetryProtocol.Implementation;
+using OpenTelemetry.Exporter.OpenTelemetryProtocol.Implementation.ExportClient;
+#if NETSTANDARD2_1
+using Grpc.Net.Client;
+#endif
+using OtlpCollector = Opentelemetry.Proto.Collector.Trace.V1;
+
+namespace OpenTelemetry.Exporter
+{
+    internal static class OtlpExporterOptionsExtensions
+    {
+#if NETSTANDARD2_1
```

Comment:
I think now that we have NET5.0 target all of these `#if NETSTANDARD2_1` need to be updated to:

`#if NETSTANDARD2_1 || NET5_0_OR_GREATER`

Reply:
Not here. According to the list of [.NET implementations](https://docs.microsoft.com/en-us/aspnet/core/grpc/netstandard?view=aspnetcore-5.0) there is no `Grpc.Net.Client` for net5.0 or greater. I think Grpc.Core.Channel should be used.